packages/web/src/components/ui/Card/Card.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styles from './Card.module.scss';
+import clsx from 'clsx';
+import Chip from '../Chip/Chip';
+import ChipGroup from '../ChipGroup/ChipGroup';
+import AvatarGroup from '../AvatarGroup/AvatarGroup';
+import VulnerablePackage from '../VulnerablePackage/VulnerablePackage';
+import { Icon } from '../Icon/Icon';
+
+export type CardProps = {
+  id: string;
+  title: string;
+  icon?: string;
+  description?: string;
+  packageTags?: {
+    featuredPackages: string[];
+    restPackages: number;
+  };
+  featuredSites?: {
+    iconList: string[];
+    numberOfUses: number;
+  };
+  vulnerablePackage?: {
+    name: string;
+    moreCount?: number;
+  };
+  variant?: 'toAll' | 'vulnerable';
+};
+
+export default function Card({
+  title,
+  icon,
+  description,
+  packageTags,
+  featuredSites,
+  vulnerablePackage,
+  variant,
+}: CardProps) {
+  return (
+    <div className={clsx(styles.card, variant && styles[variant])}>
+      <div className={styles.cardTop}>
+        <header className={styles.header}>
```

Comment:
I was wondering if it's worth using a header tag per each card? My concern is that may affect SEO some how

https://www.w3schools.com/tags/tag_header.asp

Reply:
[W3 4.3.8 The header element](https://html.spec.whatwg.org/#the-header-element)

> A [header](https://html.spec.whatwg.org/#the-header-element) element is intended to usually contain a heading (an [h1](https://html.spec.whatwg.org/#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements)–[h6](https://html.spec.whatwg.org/#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements) element or an [hgroup](https://html.spec.whatwg.org/#the-hgroup-element) element), but this is not required. The [header](https://html.spec.whatwg.org/#the-header-element) element can also be used to wrap a section's table of contents, a search form, or any relevant logos.

Probably you're right, if it is not within `<section>` tag, it is not worth it.